### PR TITLE
Fix logo in README not being visible in high-contrast mode. Fix #8160

### LINKED ISSUE #8160: Logo in README can't be fully seen
image

image

The new logo works for GitHub's dark mode, but not with GitHub's light mode. 

We may want to create a new logo with our own background color rather than using a transparent background.

### PR DESCRIPTION
**Pull request by benitaanawonah**
Added black background for dark mode and white background for light mode

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ To get started with using Wagtail, run the following in a virtual environment:
 
 ![Installing Wagtail](.github/install-animation.gif)
 
-```sh
+```bash
```
Comment (lb-) [inline README.md:61]: Please be careful to avoid unrelated changes in other sections.

Also this usage of `sh` is intentional - see https://docs.wagtail.org/en/latest/contributing/documentation_guidelines.html#when-using-console-terminal-code-blocks

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <h1 align="center">
-    <img width="343" src=".github/wagtail.svg#gh-light-mode-only" alt="Wagtail">
-    <img width="343" src=".github/wagtail-inverse.svg#gh-dark-mode-only" alt="Wagtail">
+    <img width="343" src=".github/wagtail(W).svg#gh-light-mode-only" alt="Wagtail">
+    <img width="343" src=".github/wagtail-inverse(B).svg#gh-dark-mode-only" alt="Wagtail">
```
Comment (lb-) [inline README.md:3]: Best to replace the existing images instead of adding new ones.

**Review by lb- — CHANGES_REQUESTED** [review]
A few small items of feedback, thank you for taking the time to submit these.

Once the updates are made please add a comment on this saying it's ready for another review.

I have not yet tested the images on various browser settings but these look like they are heading in the right direction.

**Comment by lb-** [comment]
@benitaanawonah  - when you get a moment, can you also please update your GitHub profile with a name - it will be easier to add you to the contributors list that way.

Also, can you please update the PR description with a reference to the bug/issue that this PR resolves.

Bonus points - add screenshots of your own validation of the new images in the various ways it can show;

* Browser normal
* Browser dark mode
* Browser normal in windows high contrast dark mode
* Browser normal in windows high contrast light mode

Thanks again for taking the time to make a contribution.

File: README.md (hunk 2 above)
Comment (benitaanawonah) [reply README.md:3]: Okay, working on it

**Comment by benitaanawonah** [comment]
This is the logo appearance in browser normal
image


This is the logo appearance in browser dark mode
image

This is the logo appearance in windows high contrast dark mode
image

This is the logo appearance in windows high contrast light mode
image

File: README.md (hunk 1 above)
Comment (benitaanawonah) [reply README.md]: It is ready for another review

**Review by thibaudcolas — CHANGES_REQUESTED** [review]
@benitaanawonah congratulations on a first pull request!

This doesn’t look far off but I think there are more changes needed so we can properly review this. It looks like you loaded the visuals in image editing software, which is fine, but if we do then they need to be exported as well as they were the last time.

Either redo the changes without loading the images in editing software – that is, change the SVG directly, or, when you export the images:

- They need to be the exact same width and height as the previous visuals
- The output needs to go through https://jakearchibald.github.io/svgomg/, and not contain any CSS class names nor styles.

**Comment by benitaanawonah** [comment]
It is ready for another review

**Comment by thibaudcolas** [comment]
Merged in c108de2bb5! In the future please try to make pull requests from other branches than `main` so they’re easier for us to merge.
